config/initializers/devise.rb

```diff
@@ -0,0 +1,290 @@
+# frozen_string_literal: true
```

Comment:
Clean out the comments from this file

Reply:
And make sure the PT link goes to the story

Reply:
Done, thanks

Reply:
Please remove the comments from this file and do not just press resolve conversation